src/sinks/aws_s3/sink.rs

```diff
@@ -82,13 +82,15 @@ impl RequestBuilder<(S3PartitionKey, Vec<Event>)> for S3RequestOptions {
                     .with_timezone(&offset)
                     .format(self.filename_time_format.as_str()),
                 None => Utc::now()
-                    .with_timezone(&chrono::Utc)
+                    .with_timezone(&Utc)
                     .format(self.filename_time_format.as_str()),
             };
 
-            self.filename_append_uuid
-                .then(|| format!("{}-{}", formatted_ts, Uuid::new_v4().hyphenated()))
-                .unwrap_or_else(|| formatted_ts.to_string())
+            if self.filename_append_uuid {
+                format!("{formatted_ts}-{}", Uuid::new_v4().hyphenated())
```

Comment:
RE previous comment: `formatted_ts` got inlined here, the previous one I mentioned should also get inlined by clippy --fix. Maybe a bug?